### DISCUSSION TITLE
Any chance nodejs bindings exist for this?

### DISCUSSION THREAD

**Discussion by orrgal1** [nanomsg/nng#1784] [Q&A] [answered]
Not really an issue, more a  question? Couldn't find anyhting..
Thanks

**Comment by gdamore** [comment]
I'm not aware of any at this point.  Would welcome a contribution here!

**Comment by pranaypratyush** [comment]
https://github.com/reqshark/nodenng
Found the above but it doesn't seem to be maintained.

**Comment by gdamore** [comment, marked as answer]
https://github.com/reqshark/nodenng is indeed pretty old.  There is a wrapper around libnanomsg, which can speak with NNG, but that's about it.
